mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -620,7 +626,7 @@ def read_prediction_metric_for_endpoint_if_exists(
         predictions = self.read_predictions(
             endpoint_id=endpoint_id, start="0", end="now", limit=1
         )
-        if predictions:
+        if predictions.data:
```

Comment:
From what I see, if this condition is False, the function returns `None`, and the metrics list will contain a `null`.

Reply:
Right, I see that you already added a TODO about that. I pushed a fix.